frontend/src/components/elements/GraphVizChart/GraphVizChart.tsx

```diff
@@ -79,20 +80,20 @@ class GraphVizChart extends React.PureComponent<PropsWithHeight> {
           ).node() as SVGGraphicsElement
           if (node) {
             this.originalHeight = node.getBBox().height
+            this.originalWidth = node.getBBox().width
           }
         })
 
-      const { height } = this.getChartDimensions()
+      const { height, width } = this.getChartDimensions()
       if (height > 0) {
         // Override or reset the graph height
         graph.height(height)
       }
-
-      // Override or reset the graph width
-      // TODO: Fix width when maximized without breaking alignment when normal.
-      // if (width > 0) {
-      //   graph.width(width)
-      // }
+      if (width > 0) {
+        // Override or reset the graph width
+        // TODO: Fix width when maximized without breaking alignment when normal.
```

Comment:
Is this something we need to handle now?

Reply:
That is solved. I will remove the comment.